dao-voting/tests/harness.rs

```diff
@@ -0,0 +1,974 @@
+#![allow(unused_variables)]
```

Comment:
These should not be left in at the top. This hides whatever mistakes you have made. Moreover, this should be split up into modules with clean imports.

Reply:
Do you mean create a module under a utils/ folder?  I tried that at one point and moved `setup()` `call_data()`, `Metadata` and `abigen!()` code there.  However when I did that I could no longer access `daovoting_mod` which I need for my test cases.  How did you handle this in fundraiser?

Reply:
I took a look at fundraisers test, but I don't see anywhere you use `contractname_mod`.  I'm able to move things over to the utils/mod file, but then I can no longer access things such as `daovoting_mod::Proposal`.  However, I may have done something wrong the first time I tried to split the file into modules, I'll give it another try.